processor/default.go

```diff
@@ -271,20 +318,23 @@ func (de *DefaultExecutor) fSendMessageEx(message, channels string, params map[s
 		}
 	}
 
+	var ts string
 	var err error
 	for _, ch := range chnls {
-		e := de.bot.Post(ch, message, atts, nil)
-		if e != nil {
-			de.command.logger.Error(e)
-			err = e
+		parent := bot.SlackMessage{}
+		parent.SetID(threadTs)
+		ts, err = de.bot.Post(ch, message, atts, &parent)
+
+		if err != nil {
+			de.command.logger.Error(err)
 		}
 	}
 
-	return "", err
+	return ts, err
 }
 
-func (de *DefaultExecutor) fSendMessage(message, channels string) (string, error) {
-	return de.fSendMessageEx(message, channels, nil)
+func (de *DefaultExecutor) fSendMessage(message, channels, threadTs string) (string, error) {
+	return de.fSendMessageEx(message, channels, threadTs, nil)
 }
```

Comment:
Hmm.. I don't like that you're limiting the method here with TS. If for example it is the first message you don't have a ts yet, then what?